### PR TITLE
Run2-hcx126 Reorder the xml files for HCAL to conform to the new idea

### PR DESCRIPTION
**Pull request by bsunanda**
Most of the scenario files are updated and tested

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/HcalCommonData/data/hcalSimNumbering/2023/hcalRecNumbering.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
```
Comment (ianna) [inline Geometry/HcalCommonData/data/hcalSimNumbering/2023/hcalRecNumbering.xml]: please, check the name hcalSimNumbering/2023/hcalRecNumbering.xml

Comment (bsunanda) [inline Geometry/HcalCommonData/data/hcalSimNumbering/2023/hcalRecNumbering.xml]: I checked that - also this happened for one hcalSimNumbering.xml file

**Comment by kpedro88** [comment]
+1

File: Geometry/HcalCommonData/data/hcalSimNumbering/2023/hcalRecNumbering.xml (hunk 1 above)
Comment (ianna) [inline Geometry/HcalCommonData/data/hcalSimNumbering/2023/hcalRecNumbering.xml]: @bsunanda - please, remove this file

Comment (bsunanda) [inline Geometry/HcalCommonData/data/hcalSimNumbering/2023/hcalRecNumbering.xml]: Doing that

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by ianna** [comment]
+1

**Comment by kpedro88** [comment]
+1

**Comment by civanch** [comment]
+1

**Comment by bsunanda** [comment]
@

**Comment by bsunanda** [comment]
@davidlange6 Please merge this version

**Comment by davidlange6** [comment]
+1

**Comment by slava77** [comment]
did this PR change geometry or was it just refactoring?

**Comment by davidlange6** [comment]
is only refactoring

**Pull request merged**